### PR TITLE
Update readme, add info about contributing

### PR DESCRIPTION
**Pull request by calda**
This PR updates the repo's `README.md` and adds info about how to contribute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 
 Lottie loads and renders animations and vectors exported in the bodymovin JSON format. Bodymovin JSON can be created and exported from After Effects with [bodymovin](https://github.com/bodymovin/bodymovin), Sketch with [Lottie Sketch Export](https://github.com/buba447/Lottie-Sketch-Export), and from [Haiku](https://www.haiku.ai).
 
-For the first time, designers can create **and ship** beautiful animations without an engineer painstakingly recreating it by hand.
+Designers can create **and ship** beautiful animations without an engineer painstakingly recreating it by hand.
 Since the animation is backed by JSON they are extremely small in size but can be large in complexity!
 Animations can be played, resized, looped, sped up, slowed down, reversed, and even interactively scrubbed.
 Lottie can play or loop just a portion of the animation as well, the possibilities are endless!
 Animations can even be ***changed at runtime*** in various ways! Change the color, position or any keyframable value!
-Lottie also supports native UIViewController Transitions out of the box!
```
Comment (calda) [inline README.md:15]: I don't know what this referred to, maybe an old API that no longer exists?

File: README.md [hunk 2]
```diff
@@ -32,6 +31,25 @@ Lottie supports [Swift Package Manager](https://www.swift.org/package-manager/),
 
 You can pull the [Lottie Github Repo](https://github.com/airbnb/lottie-ios/) and include the Lottie.xcodeproj to build a dynamic or static library.
 
+### Swift Package Manager
```
Comment (calda) [inline README.md:34]: I moved this to the top (above cocoapods and carthage)

File: README.md [hunk 3]
```diff
@@ -58,58 +76,34 @@ carthage update
...
-)
-```
-And then import wherever needed: ```import Lottie```
-
-#### Adding it to an existent iOS Project via Swift Package Manager
-
-1. Using Xcode 11 go to File > Swift Packages > Add Package Dependency
-2. Paste the project URL: https://github.com/airbnb/lottie-ios
-3. Click on next and select the project target
-4. Don't forget to set `DEAD_CODE_STRIPPING = NO` in your `Build Settings` (https://bugs.swift.org/plugins/servlet/mobile#issue/SR-11564)
```
Comment (calda) [inline README.md:87]: I removed this because I have a hunch its not necessary anymore (although didn't test it specifically)

File: README.md [hunk 4]
```diff
@@ -58,58 +76,34 @@ carthage update
...
------
-
-### Objective-C Support
+### Data collection
 
-As of 3.0 Lottie has been completely rewritten in Swift!
+The Lottie SDK does not collect any data. We provide this notice to help you fill out [App Privacy Details](https://developer.apple.com/app-store/app-privacy-details/).
 
-For Objective-C support please use Lottie 2.5.3. Alternatively an Objective-C branch exists and is still active.
+## Contributing
```
Comment (calda) [inline README.md:83]: This section is new:

File: README.md [hunk 5]
```diff
@@ -1,18 +1,17 @@
-# Lottie for iOS, macOS (and [Android](https://github.com/airbnb/lottie-android) and [React Native](https://github.com/airbnb/lottie-react-native))
```
Comment (erichoracek) [inline README.md:1]: Maybe you could link to the other platforms somewhere else (if we don't already do so)? I think being cross-platform is a key value prop of Lottie

**Review by erichoracek — APPROVED** [review]
@calda LGTM! Just a few suggestions/questions

File: README.md [hunk 6]
```diff
@@ -32,6 +31,25 @@ Lottie supports [Swift Package Manager](https://www.swift.org/package-manager/),
...
+
+``` swift
+import PackageDescription
+
+let package = Package(
+  name: "YourApp",
+  platforms: [.iOS(.v15)],
+  dependencies: [
+    .package(name: "Lottie", url: "https://github.com/airbnb/lottie-ios.git", from: "3.4.0")
+  ],
```
[the remark this answers is not in the document]
Comment (calda) [reply README.md:44]: Good point, I'm just gonna steal the instructions you wrote for Epoxy 😜

File: README.md (hunk 5 above)
Comment (calda) [reply README.md:1]: Good point, will make sure to include this somewhere

**Pull request merged**